docs/docs/connectors/your-own-website.mdx

```diff
@@ -46,34 +21,35 @@ directly, or asynchronously via a webhook.
 
 ### RestInput
 
-The `rest` channel will provide you with a REST endpoint to post messages
+The REST channel will provide you with a REST endpoint to post messages
 to and in response to that request will send back the bots messages.
```

Comment:
```suggestion
The REST channel will provide you with a REST endpoint where you can post
user messages and receive the bot's messages in response.
```

Reply:
Is this clearer?

Reply:
in response to https://github.com/RasaHQ/rasa/pull/6886#discussion_r499743854

Reply:
Yes! Way clearer